### ISSUE TITLE
Performance metrics ANNs

### ISSUE THREAD

**Issue by trrbls** [TPM034A/Q2_2023#9]
In the example on slide 19 of lecture 5, you only report the accuracy of the MLP. However, in lecture 4 you said we should always look at combinations of performance metrics. 
* Why did you choose the accuracy (as opposed to precision/recall metrics)?
* Why did you only report one metric?

Thanks in advance!

![image](https://github.com/TPM034A/Q2_2023/assets/131664634/1f59f83b-28e5-4503-a1c0-81976f22c6e3)

**Comment by sandervancranenburgh** [comment]
Good questions!
1. The performance metric(s) that is most important depends on the purpose for which the model is built. E.g. in case the model would be built to identify patients for further examination, then the False Positives are most important. Hence, evaluating the precision would be the most important metric. But in this case, no specific context is given.
2. Two metrics are reported: accuracy & cross-entropy. For classification tasks, the cross entropy is often a good generic metric because it penalises big mispredictions more than smaller ones, while accuracy is only concerned with predicting the right label (thus, a prediction of 0.51 is as good as 0.99). Thus, in this generic context, I would deem the cross entropy to be the most relevant metric to look at.  
Sander
